.azure-pipelines/generation-pipeline.yml

```diff
@@ -763,6 +768,36 @@ stages:
         - template: generation-templates/python.yml
           parameters:
             repoName: msgraph-beta-sdk-python
+
+- stage: stage_cli_v1_kiota
+  dependsOn:
+  - stage_build_and_publish_kiota
+  - stage_v1_openapi
+  condition: |
+    and
+    (
+      eq(dependencies.stage_build_and_publish_kiota.result, 'Succeeded'),
+      eq(dependencies.stage_v1_openapi.result, 'Succeeded')
+    )
+  jobs:
+  - job: cli_v1_kiota
+    steps:
+    - template: generation-templates/language-generation-kiota.yml
+      parameters:
+        language: 'shell'
+        version: ''
+        repoName: 'msgraph-cli'
+        baseBranchName : 'main'
+        branchName: 'kiota/$(v1Branch)'
+        targetClassName: "GraphClient"
+        targetNamespace: "ApiSdk"
+        customArguments: "-e '/me' -e '/me/**'"
```

Comment:
can you take this opportunity to fix the filter for GO please?

Reply:
Ok. Ronald had said he was working on it. I'll inform him that I made the change.